### PR TITLE
expose all available phone numbers

### PR DESCRIPTION
**Pull request by LBHABEBIC**
image

### CODE REVIEW

File: components/PersonView/PersonDetails.jsx [hunk 1]
```diff
@@ -80,9 +80,13 @@ const PersonDetails = ({
                 <div className="govuk-summary-list__row">
                   <dt className="govuk-summary-list__key">Phone Number</dt>
                   <dd className="govuk-summary-list__value">
-                    {Object.values(phoneNumber[0]).map((number) => (
-                      <p key={number}>{number}</p>
-                    ))}
+                    <ul>
+                      {phoneNumber.map(({ phoneNumber, phoneType }) => (
+                        <li className="govuk-list" key={phoneNumber}>
```
Comment (sirLisko) [inline components/PersonView/PersonDetails.jsx:85]: `govuk-list` should be on the `ul`, see reference at https://design-system.service.gov.uk/styles/typography/

**Review by naomigreen — APPROVED** [review]

**Comment by naomigreen** [comment]
Looks great! Just Luca's comment to update

File: components/PersonView/PersonDetails.jsx (hunk 1 above)
Comment (LBHABEBIC) [reply components/PersonView/PersonDetails.jsx]: done now. doesn't seem to make any difference.

**Review by sirLisko — APPROVED** [review]
Looks good 👍

File: components/PersonView/PersonDetails.jsx (hunk 1 above)
Comment (sirLisko) [reply components/PersonView/PersonDetails.jsx]: yes, it's more because it's cleaner to set it to the parent rather than in all the children :)

**Pull request merged**